config/application.rb

```diff
@@ -48,7 +48,6 @@ class Application < Rails::Application
     # Enable the asset pipeline
     config.assets.enabled = true
 
-    # TODO: Move to asset pipeline enabled by moving assets from vmdb/public to vmdb/app/assets
```

Comment:
@jrafanie ok, i'll get back comment until it's done.
@martinpovolny, @dclarizio do you know how is the status of moving assets pipeline?